app.py

```
@@ -540,20 +515,18 @@ def insert_user(user_id, table, cursor):  # 初始化（新增）傳入該ID的
     # Sending a PUT request to star the repository
     # pylint: disable-next = missing-timeout
     response = requests.put(star_url, headers=headers)
-    print(response.text)
     # Checking the response status and returning an appropriate message
     if response.ok:
         print(f"Successfully starred {repo_owner}/{repo_name}! {response}")
         connection, cursor = link_sql()  # SQL 會話
         if not user_id_exists(
             discord_user["id"], "user", cursor
-        ):  # 該 uesr id 不在user表格內，插入該筆使用者資料
+        ):  # 該 user id 不在user表格內，插入該筆使用者資料
```

Ruff

Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)